test/unit/chruby_reset_test.sh

```diff
@@ -15,10 +15,13 @@ function test_chruby_reset_hash_table()
 		assertEquals "did not clear the path table" \
 			     "" "$(hash)"
 	elif [[ -n "$BASH_VERSION" ]]; then
+		export LANG=en_US.UTF-8
```

Comment:
Note: tested on my machine, setting system language to Chinese. Without this change, the error message contains Chinese.